homeassistant/components/knx/__init__.py

```diff
@@ -80,14 +78,11 @@
 CONFIG_SCHEMA = vol.Schema(
     {
         DOMAIN: vol.All(
-            # deprecated since 2021.4
-            cv.deprecated(CONF_KNX_CONFIG),
```

Comment:
I'd leave this there with `cv.deprecated("config_file"),` so users who update late get a decent error message.

Reply:
Makes sense. I readded it :)